src/reporter/plugin-host.js

```diff
@@ -61,6 +62,10 @@ export default class ReporterPluginHost {
             'span syntax-comment':    str => this.chalk.grey.bold(str),
             'span syntax-invalid':    str => this.chalk.inverse(str),
 
+            [`span ${DIFF_COLORS.DIFF_FILLER}`]:  str => this.chalk.gray(str),
```

Comment:
As far as I understand the `FILLER` means that the part is equal or not modified. Not sure about name.